doc/API.md

```diff
@@ -722,6 +723,21 @@ lists the status of all files.
 }
 ```
 
+#### /renter/prices [GET]
+
+lists the estimated prices of performing various network operations.
+
+###### JSON Response [(with comments)](/doc/api/Renter.md#json-response-4)
+```javascript
+{
+  "downloadterabyte": "1234", // hastings
+  "formcontracts":    "1234", // hastings
+  "storageterabyte":  "1234", // hastings
+  "uploadterabyte":   "1234", // hastings
```

Comment:
it's important here because we're normalizing to unusual units. Sia has a defined set of units that it uses for everything - hastings, bytes, blocks. But here we are departing from that, so I felt that we should be explicit. Especially because it's pretty arbitrary.